### ISSUE TITLE
Unable to run code in the Quick Start

### ISSUE THREAD

**Issue by lcw** [exanauts/ExaPF.jl#138]
In reviewing your JuliaCon submission, https://github.com/JuliaCon/proceedings-review/issues/72, I have been exercising you code.  I have been unable to run the code described in the quick start, both on the master branch and on the develop branch.

First I started making changes like the following
```diff
diff --git a/docs/src/quickstart.md b/docs/src/quickstart.md
index a9fc2c7..491fe75 100644
--- a/docs/src/quickstart.md
+++ b/docs/src/quickstart.md
@@ -69,6 +69,7 @@ in a few lines of code.
 We first instantiate a `PolarForm` object to adopt a polar formulation
 as a model:
 ```julia-repl
+julia> using KernelAbstractions
 julia> polar = PolarForm(pf, CPU())
 
 ```
@@ -90,7 +91,7 @@ Hence, the algorithm requires the following elements:
 
 that translate to the Julia code:
 ```julia-repl
-julia> physical_state = get(polar, PhysicalState())
+julia> physical_state = get(polar, ExaPF.PhysicalState())
 julia> jx = ExaPF.init_ad_factory(polar, physical_state)
 julia> linear_solver = DirectSolver()
```
but I ran into the issue that `ERROR: UndefVarError: init_ad_factory not defined`.  I am stuck here.

p.s. it looks like the nested code blocks are messing up the formatting, not sure how to fix that.

**Comment by frapac** [comment]
Dear Lucas,
thanks for pointing out this issue. 
we are sorry for the inconvenience. ExaPF has been refactored a lot for the last three months, and we haven't proceed to any release since December (hence the slightly deprecated doc). 

We should be ready to do a release `v0.5.0` in the next few days. At least, that will ensure that everything is up to date on master. The documentation has also been updated recently: #134 

I was not aware that the nested code blocks are messing up the formatting. I will investigate that.

**Comment by lcw** [comment]
> I was not aware that the nested code blocks are messing up the formatting. I will investigate that.

This was only for my diff above.

**Comment by frapac** [comment]
Closing as this issue was solved in #140
